Functions/xASL_stat_GetROIstatistics.m

```diff
@@ -592,13 +597,6 @@
 					x.S.DAT_CoV_PVC0(SubjSess,iROI) = xASL_stat_ComputeSpatialCoV(DataIm, CurrentMask, 0, 0);
 				end
             else
-                % Start with defaults

```

Comment:
So I would put the initialization back to this location, if the masks are empty it should still initialize a NaN, otherwise, there will be zeros for subject-ROI combinations with an empty mask.

Reply:
OK